source/_posts/2016-02-15-multi-room-audio-with-snapcast.markdown

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title: "Multi-room audio with Snapcast, Mopidy, and Home Assistant"
+date: 2016-02-15 17:10:56 -0500
+author: happyleavesaoc
+comments: true
+categories: How-To
+---
+
+Multi-room audio can be achieved by having a computer attached to speakers in every room. On each computer, services run to play and/or control the audio. With this DIY approach, the kind of computer and speakers is very much up to you. It could be your desktop computer with attached powered speakers, your HTPC hooked up to your TV and receiver, a Raspberry Pi with Amp or DAC, or even an Android device.
```

Comment:
I think that the post needs an introduction to what the post will be about. 

> This tutorial will help you get started using multi-room audio so that you can wander your house and never miss a beat!